torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -209,6 +209,9 @@ void CudaPrinter::visit(const Intrinsics* v) {
   if (returnType == ScalarType::Half || returnType == ScalarType::Float) {
     func_name = func_name + "f";
   }
+  if (v->op_type() == IntrinsicsOp::kFabs && is_integral(returnType)) {
```

Comment:
does nvrtc not handle std::abs or ::abs? All this exp/expf is so last decade (and is not used in eager part of the codebase at all).

Reply:
I think it does, actually.  I don't know why we went the `+ "f"` route, except that maybe the old fuser uses it?  I'll take a note to use `std::` in a follow-up.